src/docs/01_introduction_and_goals.adoc

```diff
@@ -1,10 +1,9 @@
 [[section-introduction-and-goals]]
 == Introduction and Goals
 
-This is project viade_en2a and it will consist of a SOLID app.
+This is the project viade_en2a and it will consists of a route managing application following the https://github.com/solid/solid-spec[SOLID specification] envisioned by https://www.w3.org/People/Berners-Lee/card#i[Tim Berners Lee]
```

Comment:
Please check the spelling of consist.

Reply:
You also need a dot at the end of the sentence:
... it will consists of ... [Tim Berners Lee]
... it will consist of ... [Tim Berners Lee].